content/en/docs/ops/best-practices/image-signing-validation/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Image Signing And Validation
+description: Describes how to use image signatures to verify the provenance of Istio images.
+weight: 35
+aliases:
+    - /docs/ops/best-practices/image-signing-validation/
```

Comment:
Not sure we need an alias for a new page.

Reply:
Removed